README.md

```diff
@@ -1,38 +1,50 @@
 # DenoDB database storage adapter for grammY
 
-Database storage adapter that can be used to [store your session data](https://grammy.dev/plugins/session.html) via [DenoDB](https://github.com/eveningkid/denodb) when using sessions.
+Database storage adapter that can be used to
```

Comment:
The config file `deno.json` allows you to preserve the convention of having one sentence per line. This rule helps with having readable diffs when the markdown changes, as whole paragraphs don't reflow when adding a single word. Please stick with it.

Reply:
Weird, in my VSCode i see it's splitted
![image](https://user-images.githubusercontent.com/42675886/148686761-7657a2a9-8d7e-4662-915f-ea53ff747fbe.png)

Reply:
Sorry i think i misunderstood your comment.

Reply:
The point is that markdown docs should be written one sentence per line, for the reason outlined above. That's also how it was done here before.

By default, `deno fmt` does not allow this. It always reflows all paragraphs. We need to specify the `"preserve"` property is the Deno config file, and format the docs according to that config file. As you did not specify `deno.json` in `./.vscode/settings.json`, VSCode did not pick up the updated config, and caused all markdown docs to be formatted without preserving the match between lines and sentences.

What I meant was just that these changes should be reverted and all sentences should be on their own lines again.

Reply:
I've added formatter only for typescript, so `deno fmt` no more formatting other from `.ts` files.